state/state_test.go

```diff
@@ -30,35 +32,45 @@ func setupTestCase(t *testing.T) (func(t *testing.T), dbm.DB, *State) {
 	return tearDown, stateDB, state
 }
 
+// TestStateCopy tests the correct copying behaviour of State.
 func TestStateCopy(t *testing.T) {
 	tearDown, _, state := setupTestCase(t)
 	defer tearDown(t)
+	// nolint: vetshadow
 	assert := assert.New(t)
 
 	stateCopy := state.Copy()
 
 	assert.True(state.Equals(stateCopy),
-		cmn.Fmt("expected state and its copy to be identical. got %v\n expected %v\n", stateCopy, state))
+		cmn.Fmt(`expected state and its copy to be identical. got %v\n expected %v\n`,
```

Comment:
these newlines feel unecesary ...

Reply:
It depends on whether we want to stick to the 100 line length limit.